nix/README.rst

```diff
@@ -32,7 +32,9 @@ Once you have a working Nix installation use::
 in the base :term:`FiPy` directory to install :term:`FiPy` with
 Python 2. To install with Python 3, use::
 
-    $ nix-shell --pure nix/py3.nix
+    $ nix-shell --pure --argstr python_version 37
```

Comment:
Recommend 

> ```bash
> $ nix-shell --pure --argstr python_version 36
> ```
> to use Python version 3.6.

since none of us can successfully build with `37` at the moment.